### PR TITLE
Spiral scan from barnaby

### PR DESCRIPTION
**Pull request by TheoLaudatQM**
I extracted the use-case from the barnaby branch and put it under Quantum-Dots to keep his other files

### CODE REVIEW

File: Quantum-Control-Applications/Quantum-Dots/Use Case 1 - Fast 2D Scans/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Rather than Raster scans we used a spiral pattern to navigate to every pixel across the two-dimensional window. 
+
+|  ![Spiral_Scan](./_images/Spiral_Scan.png) | ![Spiral_Waveform](./_images/Spiral_Waveform.png)  |
+|-------------------------------------------:|:---------------------------------------------------|
+
+For a scan of $N \times N$  pixels where time $\tau$ is spent at each pixel, the slowest frequency needed for the spiral 
+pattern is $f=1/(2N\tau)$ whereas for a Raster scan it was $f=1/(N^2\tau)$.
+
+Thus allowing much greater resolutions and/or time spent at each pixel before the high pass filtering effect becomes an issue. 
+Shown below is the effect of a high pass filter with a time constant of 200µs on the waveforms needed to create 101x101 raster and spiral scans with a measurement time of 1us.
```
Comment (yomach) [inline Quantum-Control-Applications/Quantum-Dots/Use Case 1 - Fast 2D Scans/README.md:92]: The figure below is not very clear (to those who don't know it).
I would add a few words describing it...
I would also recreate the figures without analog output 3 (it just takes away focus, give out no information) and would rename the captions:
Ideal V_rg
Ideal V_lg
Simulated filtered V_rg
Simulated filtered V_lg

**Review by yomach — COMMENTED** [review]
Looks good besides my comment.

File: Quantum-Control-Applications/Quantum-Dots/Use Case 1 - Fast 2D Scans/README.md (hunk 1 above)
Comment (TheoLaudatQM) [reply Quantum-Control-Applications/Quantum-Dots/Use Case 1 - Fast 2D Scans/README.md]: Very good point, thanks !
I tried to improve it so please tell me if something remains unclear

**Review by yomach — APPROVED** [review]
I think it looks great now!

**Pull request merged**